Description: Remove use of git in doxygen.
Author: Shengqi Chen <harry@debian.org>
Forwarded: not-needed
Last-Update: 2025-03-03
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/docs/libwget.doxy.in
+++ b/docs/libwget.doxy.in
@@ -718,7 +718,7 @@
 # by doxygen. Whatever the program writes to standard output is used as the file
 # version. For an example see the documentation.
 
-FILE_VERSION_FILTER    = "/bin/sh -c 'git log --pretty=\"format:%ci, (build %h)\" -1' \"${1}\""
+FILE_VERSION_FILTER    = "/bin/sh -c 'echo \"${DEB_VERSION}\""
 
 # The LAYOUT_FILE tag can be used to specify a layout file which will be parsed
 # by doxygen. The layout file controls the global structure of the generated
